packages/server/modules/workspaces/services/invites.ts

```diff
@@ -505,14 +509,14 @@ export const validateWorkspaceInviteBeforeFinalizationFactory =
       userId: finalizerUserId
     })
     if (!workspace) {
-      throw new InviteFinalizingError(
+      throw new WorkspaceNotFoundError(
```

Comment:
Some of these changed errors may break FE code that relies on a specific error code being returned, not sure, but if not needed I would hold off against changing already BaseApp-extending error types to different ones

Reply:
Or you should do a search of the code value inside `frontend-2` to see if its referenced anywhere